modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFRequest.macro

```diff
@@ -13,6 +13,18 @@ definition {
 			locator1 = "PRMMDFRequest#ADD_BUTTON");
 	}
 
+	@summary = "Change MDF status"
+	macro changeMDFStatus {
+		var key_mdfName = "${mdfName}";
```

Comment:
hi @mateussandes23, could you please replace the variable `mdfName` for `MDFName`? Thanks!